src/core/namespace.c

```diff
@@ -1310,7 +1301,7 @@ static int mount_image(
                         image_policy,
                         host_os_release_id,
                         host_os_release_version_id,
-                        host_os_release_level,
+                        host_os_release_sysext_level ?: host_os_release_confext_level,
```

Comment:
this doesn't look right. `host_os_release_sysext_level` now always points to the *sysext* level, thus if you attach a confext this way you'll end up comparing the wrong field.

Reply:
yeah this never worked